### ISSUE TITLE
demo for squeeze

### ISSUE THREAD

**Issue by Yilin-Yu** [shaido987/multi-dim-baselines#1]
Hi,

I try to understand the algorithm squeeze and someone recommend your project. But I can't understand the detail of how each function works, especially the meaning of the input of def squeeze(df, attributes, delta_threshold=0.9, debug=False), line 124 in squeeze.py. Could you provide a demo in the future? Thank you.

Best,
YYL

**Comment by shaido987** [comment]
Hi,

you can take a look at the `evaluate.py` file to see an example of how the algorithms are run. The data is available at https://github.com/NetManAIOps/Squeeze. 

You should also note that I'm still in the process of trying to reproduce the results given in the paper. As of right now, the results of my implementation are worse (especially on the harder datasets). So please tell me if you find any inconsistencies with the paper or other issues.

Best regards

**Comment by willxi** [comment]
hi did the squeeze code run successfully?

**Comment by shaido987** [comment]
Yes, it's possible to run it however, I haven't managed to replicate the results in the paper. You can see here: https://github.com/NetManAIOps/Squeeze/issues/3 for some related discussion on parameter settings and details not included in the paper itself.
